Assets/Scripts/EnemyController.cs

```diff
@@ -10,25 +10,29 @@ public class EnemyController : MonoBehaviour {
 
     private Rigidbody2D enemyBody;
     private bool knockback = false;
+    // -1 is left
```

Comment:
we need to abstract the direction = -1 thing more.... leave this here for now though, just something to keep in mind